addons/crm/views/crm_lead_views.xml

```diff
@@ -567,10 +568,9 @@
                             <ul class="oe_kanban_colorpicker" data-field="color"/>
                         </t>
                         <t t-name="kanban-box">
-                            <t t-set="lost_ribbon" t-value="!record.active.raw_value and record.probability and record.probability.raw_value == 0"/>
+                            <t t-set="lost_ribbon" t-value="record.is_lost.raw_value"/>
```

Comment:
Would say we don't really need a variable anymore, we could just make it conditioned based on "record.is_lost.raw_value", right?